docs/storage-node-configuration.md

```diff
@@ -146,10 +146,10 @@ morph:
 | `rpc_endpoint`  | list of [endpoint descriptions](#rpc_endpoint-subsection) |               | Array of endpoint descriptions.                                                                                                 |
 
 ## `rpc_endpoint` subsection
-| Parameter  | Type     | Default value | Description                                                                                                                                       |
-|------------|----------|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
-| `address`  | `string` |               | _WebSocket_ N3 endpoint.                                                                                                                          |
-| `priority` | `int`    | `0`           | Priority of an endpoint. Endpoint with a higher priority has more chance of being used. Endpoints with equal priority are iterated over randomly. |
+| Parameter  | Type     | Default value | Description                                                                                                                                                                   |
+|------------|----------|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `address`  | `string` |               | _WebSocket_ N3 endpoint.                                                                                                                                                      |
+| `priority` | `int`    | `1`           | Priority of an endpoint. Endpoint with a higher priority (lower configuration value) has more chance of being used. Endpoints with equal priority are iterated over randomly. |
```

Comment:
As I understand priority - "Endpoint with a higher priority `must be / will be` used until it's healthy" rather than `has more chance`

Reply:
that sentence means that it is possible to have more than one endpoint with the same priority; they are shuffled so it is not guaranteed that the first gonna be used

moreover, we are [planning](https://github.com/nspcc-dev/neofs-node/issues/1615) to have some mechanism that would try to return to the most prioritized endpoint but it is not implemented yet. so it is all about the probability

Reply:
Weighted probability